tests/integration/testlib/env.py

```diff
@@ -38,3 +38,15 @@ def is_k8s():
 
 def is_el8():
     return exec_cmd("rpm -E %{?rhel}".split())[1].strip() == "8"
+
+
+def nm_libreswan_version_int():
+    version_str = exec_cmd(
```

Comment:
check=True ?

Reply:
Other than the nit above, the patch looks good to me.

Reply:
Added `check=True`.